buildstockbatch/localdocker.py

```diff
@@ -114,7 +114,9 @@ def run_building(cls, project_dir, buildstock_dir, weather_dir, docker_image, re
             if not os.path.exists(dir_to_make):
                 os.makedirs(dir_to_make)
 
-        osw = cls.create_osw(cfg, n_datapoints, sim_id, building_id=i, upgrade_idx=upgrade_idx)
+        output_dir = '../../../run'
+        hpxml_path = '../../../run/in.xml'
+        osw = cls.create_osw(cfg, n_datapoints, sim_id, building_id=i, hpxml_path=hpxml_path, output_dir=output_dir, upgrade_idx=upgrade_idx)
```

Comment:
We need to pass in `output_dir` and `hpxml_path` only when using the `residential_hpxml` workflow generator.